airbyte-workers/build.gradle

```diff
@@ -40,7 +40,7 @@ dependencies {
     implementation 'com.auth0:java-jwt:3.19.2'
     implementation 'io.fabric8:kubernetes-client:5.12.2'
     implementation libs.guava
-    implementation (libs.temporal.sdk) {
+    implementation(libs.temporal.sdk) {
```

Comment:
curious, why are these changes happening?

Reply:
It's part of the formatting rules we have defined, which I have configured to run on any file that has been saved.  And this is consistent with [other build.gradle file formats](https://github.com/airbytehq/airbyte/blob/31d7eb39c5dd1b078654b47cec3833ec0d1e99c4/airbyte-server/build.gradle#L20-L28).

My guess is that the `gradle format` task only targets java files, which is why these slight formatting differences only fail PR builds.